Dockerfile

```diff
@@ -6,6 +6,8 @@ MAINTAINER Jupyter Project <jupyter@googlegroups.com>
 
 USER root
 
+RUN apt-get update
```

Comment:
You should only need to do the apt-get update once at the beginning of your
Dockerfile, so just before your first apt-get install.

It makes sure that your apt source.lists are uptodate (they weren't which
is why you were getting a boat load of 'unable to fetch' errors).



On 13 September 2015 at 10:23, Chris Snow <notifications@github.com> wrote:

> In Dockerfile
> <https://github.com/jupyter/docker-demo-images/pull/69#discussion_r39345631>
> :
>
> > @@ -6,6 +6,8 @@ MAINTAINER Jupyter Project <jupyter@googlegroups.com>
> >
> >  USER root
> >
> > +RUN apt-get update
>
> Do you mean add it before each apt-get install command? E.g.
>
> RUN apt-get update && apt-get install -y julia libnettle4 && apt-get clean
>
> What is your rationale for doing this? Is it to minimise the size of the
> docker image?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jupyter/docker-demo-images/pull/69/files#r39345631>.
>